buildSrc/src/main/groovy/org/elasticsearch/gradle/BuildPlugin.groovy

```diff
@@ -711,26 +709,13 @@ class BuildPlugin implements Plugin<Project> {
                 // after the doFirst added by the info plugin, and we can override attributes
                 JavaVersion compilerJavaVersion = ext.get('compilerJavaVersion') as JavaVersion
                 jarTask.manifest.attributes(
+                        // TODO: remove using the short hash
+                        'Change': ((String)ext.get('gitRevision')).substring(0, 7),
```

Comment:
If the intention is to remove the short hash, why not just remove it here, now?

Reply:
To be clear, I want to keep the hash in the product build, so that it is in the manifest and displayed at startup and in the main REST action. The reason that I do not change it here is because, well, it's a separate concern and I like one commit per concern. Practically speaking, with that change, we should change the field names used in the Build.java (from `shortHash` to `hash`. Doing that change here makes this change larger (and again, about two things instead of one).